### DISCUSSION TITLE
Watchlist

### DISCUSSION THREAD

**Discussion by barbaros** [ghostfolio/ghostfolio#821] [Ideas]
Just found your project today via awesome self hosted github project and I should say it looks really nice. I didn’t had a chance to self host it yet but had a look at the demo. I was wondering is there an option to create watchlists and add stocks etc. To the watchlists? It’s really common feature for most of the stocks app so if you don’t have this feature it would be great to have it.

Thanks!

**Comment by dtslvr** [comment]
Thanks for starting a new discussion, @barbaros. There are no watchlists yet, but I like the idea.

**Reply by dtslvr** [reply]
I have created a few issues:
* https://github.com/ghostfolio/ghostfolio/issues/4564
* https://github.com/ghostfolio/ghostfolio/issues/4584
